src/components/with-text.js

```diff
@@ -41,8 +41,12 @@ import translateMapping from '../lib/translate-mapping';
  *		<input placeholder={props.placeholder} />
  *	)
  */
-export const withText = mapping => Child => (props, context) => {
-	let map = typeof mapping==='function' ? mapping(props) : mapping;
-	let translations = translateMapping(map, context.intl);
-	return <Child {...props} {...translations} />;
-};
+export function withText(mapping) {
+	return function withTextWrapper(Child) {
```

Comment:
can drop this name since it gets uglified out

Reply:
Would it be worth keeping so that tools that pull in the `jsnext:main` can use it?  That's a good chunk of why we started this in the first place.  The names get mangled too by the uglifier.